lnclient/ldk/ldk.go

```diff
@@ -224,6 +224,10 @@ func NewLDKService(ctx context.Context, cfg config.Config, eventPublisher events
 		"duration": math.Ceil(time.Since(syncStartTime).Seconds()),
 	}).Info("LDK node synced successfully")
 
+	// backup channels after successful startup
+	// TODO: consider removing after all hubs are updated past 1.9.1
+	ls.backupChannels()
```

Comment:
don't you think it's good to keep this on bootup anyway? we write new backups anyway.

Reply:
maybe it's good, yeah